From: SeongJae Park <sj@kernel.org>
Date: Thu, 19 Jun 2025 11:26:09 -0700
Subject: [PATCH] ==== memcg_path memleak fix ====

mm/damon: fix memory leak in memcg_path sysfs file

Users can leak memory by repeatedly writing a string to DAMOS sysfs
memcg_path file.  Fix it (patch 1) and add a selftest (patch 2) to avoid
reoccurrance of the bug.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_5s485ge3 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_5s485ge3

diff --git a/hkml_cv_bogus/hkml_cv_bogus_5s485ge3 b/hkml_cv_bogus/hkml_cv_bogus_5s485ge3
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

